src/zarr/api/synchronous.py

```diff
@@ -155,8 +157,9 @@ def save_group(
     )
 
 
-def tree(*args: Any, **kwargs: Any) -> None:
-    return sync(async_api.tree(*args, **kwargs))
+@deprecated("Use Group.tree instead.")
+def tree(grp: Group, expand: bool | None = None, level: int | None = None) -> Any:
+    return sync(async_api.tree(grp._async_group, expand=expand, level=level))
 
```

Comment:
```suggestion
tree.__doc__ = async_api.tree.__doc__
```
I *think* this should fix the missing docstring at https://zarr--2537.org.readthedocs.build/en/2537/_autoapi/zarr/convenience/index.html#zarr.convenience.tree

Reply:
Hmmmm, we already have that! 

https://github.com/zarr-developers/zarr-python/blob/0cca6b061e1f98594b206c69ecfc9f1c8972b4ea/src/zarr/api/synchronous.py#L302

Reply:
Huh, odd

Reply:
Oh I think it might be becauase `autoapi` doesn't actualy import code to scrape docstrings. Anyway, not worth fixing in this PR